java/src/main/kotlin/io/spine/validation/java/generate/ValidationCodeInjector.kt

```diff
@@ -129,49 +129,27 @@ private fun MessagePsiClass.implementValidatableMessage() {
 }
 
 /**
- * Declares `validate()` method in this [MessagePsiClass].
+ * Declares the `validate(parentPath, parentName)` method in this [MessagePsiClass].
  *
- * This is a `public` implementation of [ValidatableMessage.validate] with
- * [Override] annotation. The actual constraints are contained in its
- * private overloading, that accepts the field path.
+ * This method implements the logic for verifying that the message’s constraints are met.
+ * It takes the parent path and name as arguments to preserve this information for cases
+ * when in-depth validation takes place. This data is used to construct constraint violations.
  *
- *  @see declarePrivateValidateMethod
+ * In typical (top-level) validations, the [ValidatableMessage.validate] method is called,
```

Comment:
I don't think "validation" exists in plural. Can we rephrase this as "In typical use cases of validating the top-level messages ..."?